server/lorax_server/models/model.py

```diff
@@ -86,6 +86,10 @@ def info(self) -> InfoResponse:
     @property
     def sliding_window_blocks(self) -> Optional[int]:
         return None
+    
+    @property
+    def supports_embeddings(self) -> bool:
```

Comment:
Conversely, what happens when someone tries to use the BERT model for text generation?